components/Common/WarrantyButtonSvg/WarrantyButtonSvg.tsx

```diff
@@ -0,0 +1,217 @@
+import { FaqDataCategory } from "@lib/types";
+
+interface WarrantyButtonSvgProps {
+  category: FaqDataCategory;
+  color: "white" | "gray";
+}
+
+const WarrantyButtonSvg = ({ category, color }: WarrantyButtonSvgProps) => {
+  const getWarrantyButtonSvgColor = () => {
+    switch (color) {
+      case "white":
+        return "white";
+      case "gray":
+        return "#838EA4";
+      default:
+        return null;
```

Comment:
@oscarcorellan null is not a valid color, let's return instead an empty string
```suggestion
        return "";
```